dataland-frontend/tests/e2e/fixtures/frameworks/gdv/GdvPreparedFixtures.ts

```diff
@@ -87,12 +87,23 @@ function modifyPreparedFixturesPartTwo(newFixture: FixtureData<GdvData>): void {
   if (newFixture.t.allgemein?.anleihen?.sustainibilityLinkedDebt) {
     newFixture.t.allgemein.anleihen.sustainibilityLinkedDebt = YesNo.Yes;
   }
+  if (newFixture.t.allgemein?.esgBerichte?.aktuelleBerichte) {
```

Comment:
Why are these only conditionally added.

Reply:
Cannot set a property that is potentially undefined in TS.
Therefore I need to make sure the property exists before I can set it.

However it is implicitly expected that all of these properties are not null, because I use this function on a gdv-dataset without null values.

Reply:
I see. Ok